app/services/testdata/Phase3TestsInvitedStatusGenerator.scala

```diff
@@ -41,7 +47,27 @@ trait Phase3TestsInvitedStatusGenerator extends ConstructiveGenerator {
   val p3TestService: Phase3TestService
   val gatewayConfig: LaunchpadGatewayConfig
 
-  def generate(generationId: Int, generatorConfig: GeneratorConfig)(implicit hc: HeaderCarrier, rh: RequestHeader) = {
+  def generate(generationId: Int, generatorConfig: GeneratorConfig)
+              (implicit hc: HeaderCarrier, rh: RequestHeader): Future[DataGenerationResponse] = {
+
+    val oneDayInMillis = 1000 * 60 * 60 * 24
+    val launchpad = LaunchpadTest(
+      interviewId = 12345,
+      usedForResults = true,
+      testUrl = "http:///www.fake.url",
+      token = UUID.randomUUID().toString,
+      candidateId = UUID.randomUUID().toString,
+      customCandidateId = "FSCND-123",
+      invitationDate = generatorConfig.phase3TestData.flatMap(_.start)
+        .getOrElse(DateTime.now().withDurationAdded(oneDayInMillis, -1)),
```

Comment:
This can be .plusDays(1) instead of .withDurationAdded, then we can also bin off oneDayInMillis.